setup.cfg

```diff
@@ -63,9 +63,12 @@ install_requires =
     matchcode-toolkit == 7.0.0
     purl2vcs == 2.0.0
     univers == 30.12.1
-    scancodeio @ git+https://github.com/aboutcode-org/scancode.io.git@3eeb0c040c6077d773680b59da6fa283ba1500bf
+    scancodeio @ git+https://github.com/aboutcode-org/scancode.io@bumb-hashid
```

Comment:
@keshav-space are you sure you branch also covers the commit above?
@JonoYang ping

Reply:
@keshav-space https://github.com/aboutcode-org/scancode.io/pull/1479 is merged now. Can you update your PR and also take into account that: https://github.com/aboutcode-org/scancode.io/commit/3eeb0c040c6077d773680b59da6fa283ba1500bf and then sync with @JonoYang  ?

Reply:
@pombredanne https://github.com/aboutcode-org/scancode.io/commit/3eeb0c040c6077d773680b59da6fa283ba1500bf belongs to https://github.com/aboutcode-org/scancode.io/pull/1466 which is already merged. I will update the PR to use the scancode.io@main